pkg/csource/csource_test.go

```diff
@@ -285,7 +285,7 @@ syscall(SYS_csource8, /*num=*/(intptr_t)-1);
 				target:    test.target,
 				sysTarget: targets.Get(test.target.OS, test.target.Arch),
 			}
-			calls, _, err := ctx.generateProgCalls(p, false)
+			calls, _, err := ctx.generateProgCalls(p, false, false)
```

Comment:
Can you please add a comment that we disable comment generation here to make the test more straightforward etc?